tests/playwright/sanity/modules/nested-accordion/helper.js

```diff
@@ -0,0 +1,109 @@
+const { expect } = require( '@playwright/test' );
+
+/**
+ * Set Nested Accordion Title Tag (H1-H6,div,span,p)
+ *
+ * @param {string} optionToSelect          - value of select option i.e. h1,h2,h3,h4,h5,h6,div,span,p
+ * @param {string} nestedAccordionWidgetId - id of the nested accordion widget
+ * @param {Object} editor
+ * @param {Object} page
+ * @return {Promise<void>}
+ */
+async function setTitleTextTag( optionToSelect, nestedAccordionWidgetId, editor, page ) {
+	const frame = editor.getPreviewFrame();
+	await editor.selectElement( nestedAccordionWidgetId );
+	await page.selectOption( '.elementor-control-title_tag .elementor-control-input-wrapper > select', optionToSelect );
+	await frame.waitForLoadState( 'load' );
+}
+
+/**
+ * Take a Screenshot of this Widget
+ *
+ * @param {string} fileName
+ * @param {Object} locator
+ * @return {Promise<void>}
+ */
+async function expectScreenshotToMatchLocator( fileName, locator ) {
+	expect.soft( await locator.screenshot( {
+		type: 'png',
+	} ) ).toMatchSnapshot( fileName );
```

Comment:
What is causing the problem?

Reply:
@hein-obox  To fix the issue with "exception require container containers" I had to abandon using selectElement() for this particular test. Dymitro suggested we replace this because it's flaking out and causing those console exceptions "container containers required".

Clicking on the widget by ID resulted in the inner container getting selected which makes the test fail.
Workaround was to click on the item-title. But this closes the first Item. So the test failed once after pushing up this change. 

I'm not expecting this to be flaky. The failure here is because of different height like the diff screenshot I added below.

![image](https://github.com/elementor/elementor/assets/130742176/8cb4fda7-a97b-499d-ae17-851e97a3a083)